play/src/front/Components/ActionBar/AvailabilityStatus/Modals/NotificationPermissionModal.svelte

```diff
@@ -0,0 +1,35 @@
+<script lang="ts">
+    import { ConfirmationModalPropsInterface } from "../Interfaces/ConfirmationModalPropsInterface";
+    import { notificationPermissionModalVisibility } from "../../../../Stores/AvailabilityStatusModalsStore";
+    import LL from "../../../../../i18n/i18n-svelte";
+    import { analyticsClient } from "../../../../Administration/AnalyticsClient";
+    import { localUserStore } from "../../../../Connection/LocalUserStore";
+    import { iframeListener } from "../../../../Api/IframeListener";
+    import ConfirmationModal from "./ConfirmationModal.svelte";
+
+    const confirmationModalProps: ConfirmationModalPropsInterface = {
+        handleAccept: () => {
+            notificationPermissionModalVisibility.close();
+            analyticsClient.settingNotification("true");
+            Notification.requestPermission()
```

Comment:
It already exist in `MediaManager`, maybe use the same class or move this part in the `MediaManager` class